plugin_tests/cli_results_test.py

```diff
@@ -44,15 +45,15 @@ def tearDownModule():
 
 
 class CliResultsTest(unittest.TestCase):
-    def _runTest(self, cli_args=(), cli_kwargs={}, outputs={}, contains=[],
-                 excludes=[]):
-        """
-        Test a cli by calling subprocess.  Ensure that output files match a
-        sha256 and stdout contains certain phrases and excludes other phrases.
-        A value in the cli_args or cli_kwargs of 'tmp_<value>' will create a
-        temporary file and use that instead.  The same value in the output
-        list can be used to test the existence and contents of that file at the
-        end of the process.
+    def _runTest(self, cli_args=(), cli_kwargs={}, outputs={},
+                 in_process=True, contains=[], excludes=[]):
+        """Test a cli by calling runpy or subprocess.  Ensure that output
```

Comment:
I'm trying to encourage us to use Girder's convention, which is to have the doc string start on the next line.  Girder isn't wholly consistent either, but it has 1590 on the next line versus 65 on the same line...